### ISSUE TITLE
MdSlider prevents compiling with AoT

### ISSUE THREAD

**Issue by borislemke** [angular/material2#1657]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Compile project with AoT

#### What is the current behavior?
Compilation fails due to MdSlider's _tickInterval being private

#### What are the steps to reproduce?
Compile with ngc
`ngc -p src/tsconfig-aot.json`
Returns:
```
Error at ~someDevPath/src/aot/node_modules/@angular2-material/slider/slider.ngfactory.ts:106:20: Property '_tickInterval' is private and only accessible within class 'MdSlider'.
    at check (~someDevPath/node_modules/@angular/tsc-wrapped/src/tsc.js:31:15)
    at Tsc.typeCheck (~someDevPath/node_modules/@angular/tsc-wrapped/src/tsc.js:86:9)
    at ~someDevPath/node_modules/@angular/tsc-wrapped/src/main.js:33:23
    at process._tickCallback (internal/process/next_tick.js:103:7)
    at Module.runMain (module.js:606:11)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
Compilation failed
```

#### What is the use-case or motivation for changing an existing behavior?
Make compile with AoT work

#### Which versions of Angular, Material, OS, browsers are affected?
```
angular-cli: 1.0.0-beta.18
node: 6.9.1
os: darwin x64

@angular2-material 2.0.0-alpha.8-2
```

**Comment by SamiAl90** [comment]
AFAIK it's because the inputs/outputs where marked as private which won't work with AOT.
They already changed that, guess we need to wait for Alpha10.

Here https://github.com/angular/material2/pull/1636 also for tabs and outputs.

**Comment by jelbourn** [comment]
This was fixed in 833368f945ca8b39909e827ef653520e70dcf80d, which should be included in the current release (alpha.9-3)

**Comment by jelbourn** [comment]
Also note that new releases are being done on `@angular/material` instead of the previous `@angular2-material`
